common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidDevicePopManager.java

```diff
@@ -385,9 +397,6 @@ private void initialize23(@androidx.annotation.NonNull final KeyPairGenerator ke
                         KeyProperties.DIGEST_NONE,
                         KeyProperties.DIGEST_SHA1,
                         KeyProperties.DIGEST_SHA256
-                ).setEncryptionPaddings(
-                        KeyProperties.ENCRYPTION_PADDING_RSA_OAEP,
-                        KeyProperties.ENCRYPTION_PADDING_RSA_PKCS1
```

Comment:
setEncryptionPaddings is removed unconditionally.

If purposes has encrypt/decrypt, shouldn't this be kept?

Instead or purposes you can pass the use unnecessaryCryptoPurposesDisabled flag to these initialize* methods, and let each method decide its purposes.

Reply:
Are you referring to this commit? 
<img width="537" height="109" alt="image" src="https://github.com/user-attachments/assets/9f6b6b1f-846c-4649-9f7f-f944b7d28f29" />
If yes, it is removed unconditionally but added immediately conditionally based on this boolean unnecessaryCryptoPurposesDisabled